### ISSUE TITLE
accuracy 

### ISSUE THREAD

**Issue by ZlaaM** [yinguobing/cnn-facial-landmark#44]
@yinguobing 
please if you can while updating the code ( training) add the necessary statement for calculation the accuracy at each step with training and the accuracy of testing 
( example : step 1  loss = .....  , accuracy= ....)
thanks alot

**Comment by yinguobing** [comment]
Sure, this is already under consideration.

**Comment by ZlaaM** [comment]
Thank you so much for your interest

**Comment by abdou31** [comment]
@yinguobing  Please try to update the code of training and Thanks

**Comment by yinguobing** [comment]
This is a regression task and we need to define 'accuracy' before using it.
Right now mean squared error is recommended to evaluate the model performance. See latest update.
